controlplane/kubeadm/controllers/scale_test.go

```diff
@@ -303,6 +336,12 @@ func withFailureDomain(fd string) machineOpt {
 	}
 }
 
+func withAnnotation(annotation string) machineOpt {
+	return func(m *clusterv1.Machine) {
+		m.ObjectMeta.Annotations = (map[string]string{controlplanev1.DeleteControlPlaneMachineAnnotation: annotation})
```

Comment:
the `annotation` is itself the full annotation, so we don't need `controlplanev1.DeleteControlPlaneMachineAnnotation:`